src/components/NavBar/SearchBarDropdown.tsx

```diff
@@ -193,6 +201,7 @@ export const SearchBarDropdown = ({ toggleOpen, tokens, collections, hasInput, i
   }, [toggleOpen, hoveredIndex, totalSuggestions])
 
   useEffect(() => {
+    const eventProperties = { totalSuggestions, queryText }
```

Comment:
can we make sure we keep consistent snake casing for event property names, so total_suggestions, and query_text